runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxyBuilder.java

```diff
@@ -0,0 +1,55 @@
+package org.corfudb.runtime.object;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.util.serializer.ISerializer;
+
+import java.lang.reflect.Constructor;
+import java.util.UUID;
+
+/**
+ * Created by mwei on 11/11/16.
+ */
+public class CorfuCompileProxyBuilder {
+
+    @SuppressWarnings("unchecked")
+    public static <T> T getProxy(Class<T> type, CorfuRuntime rt, UUID streamID,
+                                 Object[] args, ISerializer serializer)
+        throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+        // Do we have a compiled proxy for this type?
+        Class<ICorfuSMR<T>> proxyClass = (Class<ICorfuSMR<T>>)
```

Comment:
Change the name to wrapperClass or something else since this is not the proxyClass